packages/workbench/sonar-project.js

```diff
@@ -0,0 +1,34 @@
+const argv = require('minimist')(process.argv.slice(2));
+const sonarqubeScanner = require("sonarqube-scanner");
+
+const branch = argv['branch'];
+const targetBranch = argv['target-branch'];
+const pullRequestId = argv['pull-request-id'];
+
+const sonarOptions = {
+  "sonar.organization": "ontotext-ad",
+  "sonar.projectKey": "Ontotext-AD_WB_migration_workbench",
+  "sonar.projectName": "WB Migration: workbench",
```

Comment:
Can we change the key and the name please! And same for the shared components where you also added the `WB Migration ` prefix

Reply:
done